### PR TITLE
port progress-bars page to MarkDown

### PR DESCRIPTION
**Pull request by branfosj**
If we have
```md
*(supported since: EasyBuild v4.5.0)*
```
then we fail the linter with:
```
Error: .//docs/progress-bars.md:3: MD036 Emphasis used instead of a header
```

### CODE REVIEW

File: docs/progress-bars.md [hunk 1]
```diff
@@ -1,37 +1,11 @@
 # Progress bars
 
-!!! warning
-    This page will soon replace <https://docs.easybuild.io/en/latest/Progress_bars.html>.
+(supported since: EasyBuild v4.5.0)
```
Comment (boegel) [inline docs/progress-bars.md:3]: use `*...*` to make it italic?

```suggestion
*(supported since: EasyBuild v4.5.0)*

**Review by boegel — CHANGES_REQUESTED** [review]

File: docs/progress-bars.md (hunk 1 above)
Comment (branfosj) [reply docs/progress-bars.md:3]: See https://github.com/easybuilders/easybuild-docs/pull/10#issue-1424453640

Comment (boegel) [reply docs/progress-bars.md:3]: Oh, I overlooked that... Ugh, that's a bit... weird.

Would it make sense to disable `MD036` in the linter?

Comment (branfosj) [reply docs/progress-bars.md]: I think so, as we have a legitimate reason to allow it. Done in 2de6427

**Review by boegel — APPROVED** [review]

**Pull request merged**